contracts/universalSchemes/GlobalConstraintRegistrar.sol

```diff
@@ -35,16 +35,11 @@ contract GlobalConstraintRegistrar is UniversalScheme {
         bytes32 voteToRemoveParams; // Voting parameters for removing this GC.
     }
 
-    // Struct holding the data for each organization
-    struct Organization {
-        bytes32 voteRegisterParams; // The voting parameters for adding a GC.
-        IntVoteInterface intVote; // The voting machine in which the voting takes place.
-        mapping(bytes32=>GCProposal) proposals; // A mapping from the proposal ID to the proposal itself.
-        mapping(address=>bytes32) voteToRemoveParams; // A mapping that saves the parameters for removing each GC.
-    }
+    // GCProposal by avatar and proposalId
+    mapping(address=>mapping(bytes32=>GCProposal)) public organizationProposals;
```

Comment:
organizationProposals -> organizationsProposals

Reply:
Done

Reply:
It is still organizationProposals .
It should be organizationsProposals

Reply:
It looks OK to me:  https://github.com/daostack/arc/tree/unifyProposalsAPI/contracts/universalSchemes

Reply:
It should be name organizationsProposals